docs/content/policy-cheatsheet.md

```diff
@@ -268,6 +268,98 @@ z := {"a": true, "b": "foo", "c": 4}
 object.union(y, x) == z
 ```
 
+#### Parse array of sets into nested objects
+
+This is an example of parsing an array of sets that have a known structure into nested objects.
+
+```live:rules/parse_array_to_nested_objs:query:read_only
+entity_array = [
```

Comment:
I like the idea, but IMO we should maybe minimize the example a little bit more to try and keep the cheatsheet examples short and to the point (notice that this one is significantly larger than the others in here). 

Maybe we just do like a couple array entries, shorter names (`x`, `foo`, etc) and that will also simplify the comprehensions.

Reply:
OK, will do.